integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/massindexing/MassIndexingBaseIT.java

```diff
@@ -294,6 +302,75 @@ Book.INDEX, targetTenantId(), DocumentCommitStrategy.NONE, DocumentRefreshStrate
 		backendMock.verifyExpectationsMet();
 	}
 
+	@Test
+	public void fromMappingWithoutSessionWithAllTenants() throws Exception {
+		SearchMapping searchMapping = Search.mapping( setupHolder.sessionFactory() );
+		MassIndexer indexer = searchMapping.scope( Object.class ).massIndexer( allTenantIds() );
```

Comment:
We would also need a test where tenant IDs are retrieved implicitly. I'll add that.